schema/create_raw_sales_data.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS raw_sales_data (
+    user_id VARCHAR(500),
+    username VARCHAR(500),
+    product_id VARCHAR(500),
+    product_name VARCHAR(500),
+    category_id VARCHAR(500),
+    category_name VARCHAR(500),
+    current_price VARCHAR(500),
+    sold_price VARCHAR(500),
+    sold_quantity VARCHAR(500),
+    remaining_quantity VARCHAR(500),
+    sales_date VARCHAR(500)
+);
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS raw_sales_data (
  user_id VARCHAR(500),
  username VARCHAR(500),
  product_id VARCHAR(500),
  product_name VARCHAR(500),
  category_id VARCHAR(500),
  category_name VARCHAR(500),
  current_price VARCHAR(500),
  sold_price VARCHAR(500),
  sold_quantity VARCHAR(500),
  remaining_quantity VARCHAR(500),
  sales_date VARCHAR(500)
);

```

Reply:
Apply same changes to all SQL quries. And please follow double space indentation